redisinsight/ui/src/pages/vector-search/VectorSearchPage.tsx

```diff
@@ -4,13 +4,11 @@ import { VectorSearchCreateIndex } from './create-index/VectorSearchCreateIndex'
 import { VectorSearchQuery } from './query/VectorSearchQuery'
 
 export const VectorSearchPage = () => {
-  const hasIndexes = false
+  const hasIndexes = true
```

Comment:
Question: Is that flip intentional (you had discussions with @KrumTy), or simply you forgot to stash it before pushing, since it was easier to test your changes?

Reply:
There's no particular reason for it - just wanted to make it easier for someone who does the review to be able to see the page. It really doesn't matter how we are going to merge it IMO, as the next PR should be to determine this, based on the actual database. Maybe I'm missing something, so please, elaborate if needed.